SocketRocket/SRWebSocket.m

```diff
@@ -452,10 +629,10 @@ - (void)_updateSecureStreamOptions;
         [_outputStream setProperty:sslOptions forKey:(__bridge NSString *)kCFStreamPropertySSLSettings];
     }
     
-    _inputStream.delegate = self;
-    _outputStream.delegate = self;
+    //_inputStream.delegate = self;
```

Comment:
This looks like an unintentional change. Any chance you can double check that we still assign the `delegate `for the streams?